careful.md

```diff
@@ -0,0 +1,5 @@
+## BE CAREFUL
+
+```shell
+rm /
```

Comment:
Put a note in here about how dangerous this can be, but try not to scare anyone too much.

Reply:
Fixed it! checkout the new commit!

Reply:
Mark **Down** Works